src/backend/utils/adt/agtype.c

```diff
@@ -4820,7 +4820,7 @@ Datum column_get_datum(TupleDesc tupdesc, HeapTuple tuple, int column,
  * node or edge. The function returns a pointer to a duplicated string that
  * needs to be freed when you are finished using it.
  */
-static char *get_label_name(const char *graph_name, graphid element_graphid)
```

Comment:
The element_label_id type should be int32.
Also, if element_graphid is unused, remove it.

Reply:
@CapnSpek Let's change the error message to 'label id does not exist', and remove the element_graphid parameter.

Reply:
@rafsun42 Done! Please merge.